posts/2024-02-26-2023-Rust-Annual-Survey-2023-results.md

```diff
@@ -0,0 +1,385 @@
+    </div>
+</div>
+<!-- Chart which-problems-do-you-remember-encountering end -->
+
+Respondents of the survey want the Rust maintainers to mainly prioritize fixing compiler bugs (68%), improving the runtime performance of Rust programs (57%) and also improving compile times (45%).
+
+Same as in recent years, respondents noted that compile times are one of the most important area that should be improved. However, it is interesting to note that respondents also seem to consider runtime performance to be more important than compile times.
+
+<!-- Chart how-should-work-be-prioritized start -->
+<div>
+    <div class="matrix-chart" id="how-should-work-be-prioritized" style="height:800px; width:100%;"><noscript>
+<img alt="how-should-work-be-prioritized" height="800" src="../../../images/2024-02-rust-survey-2023/how-should-work-be-prioritized.png"/>
+</noscript></div>
+    <div style="display: flex; margin-bottom: 10px;">
+        <span>[<a href="../../../images/2024-02-rust-survey-2023/how-should-work-be-prioritized.png" target="_href_" title="Download chart as PNG">PNG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/how-should-work-be-prioritized.svg" target="_href_" title="Download chart as SVG">SVG</a>]</span>
+    </div>
+</div>
+<!-- Chart how-should-work-be-prioritized end -->
+
+## Looking ahead
+
+Each year, the results of the State of Rust survey help reveal the areas that need improvement in many areas across the Rust Project and ecosystem, as well as the aspects that are working well for our community.
+
+We are aware that the survey has contained some confusing questions, and we will try to improve upon that in the next year's survey.
+If you have any suggestions for the Rust Annual survey, please [let us know](https://github.com/rust-lang/surveys/issues)!
+
+We are immensely grateful to those who participated in the 2023 State of Rust Survey and facilitated its creation. While there are always challenges associated with developing and maintaining a programming language, this year we were pleased to see a high level of survey participation and candid feedback that will truly help us make Rust work better for everyone.
+
+If you’d like to dig into more details, we recommend you to browse through the full [survey report][report].
+
+[report]: ../../../images/2024-02-rust-survey-2023/annual-survey-2023-report.pdf
+
+[a]Gracie to run this by DEI subcommittee
```

Comment:
```suggestion
[^1] Gracie to run this by DEI subcommittee
```

Reply:
This is an internal TODO I think, not something that we want to keep in the blog post :sweat_smile: